apps/account-api/src/controllers/v2/accounts-v2.controller.ts

```diff
@@ -96,6 +99,16 @@ export class AccountsControllerV2 {
     // Also makes sure it is either a login or a delegation
     const payload = await this.siwfV2Service.getPayload(callbackRequest);
 
+    // Validate claim handle transactions to prevent invalid submissions to the blockchain
+    await Promise.all(
+      payload.payloads.filter(isPayloadClaimHandle).map(async (claimHandle) => {
```

Comment:
nit/code golf; I find this more readable:
```suggestion
      payload.payloads.filter((payload) => isPayloadClaimHandle(payload)).map(async (claimHandle) => {
```